components/Forms/FormAddPlaces.js

```diff
@@ -39,7 +41,9 @@ export default function AddPlaces({ getUniqueValues }) {
     const formData = new FormData(event.target);
     const xdata = Object.fromEntries(formData);
 
-    const data = { ...xdata, activities: isChecked };
+    const data = { ...xdata, activities: isChecked, image: isImage };
```

Comment:
```suggestion
    const ownPlaceData = { ...ownPlace, activities: checkedActivties, image: uploadedImageUrl };
```
You are redeclaring data here, which can lead to errors.
Watch out for naming issues, try to make sure the variable names represent the content of the Variable :)

Reply:
Also don't forget to rename the variable data in the fetch below, so it can't be confused with the data from the activities!

`body: JSON.stringify(ownPlaceData),`